src/Header.js

```diff
@@ -39,6 +41,10 @@ class Header extends Component {
         window.location = '/';
     }
 
+    closeErrorModal() {
+        this.document.getElementById('serviceErrorModal').style.display = 'none';
```

Comment:
A thought: (Not that I necessarily want you to change it) I think we can do this using the React refs instead of this.document.

The interaction with the header from could be through an exported method.